ui/apps/platform/src/Containers/Policies/Wizard/Step5/PolicyEnforcementForm.tsx

```diff
@@ -0,0 +1,206 @@
+import React, { useState } from 'react';
+import {
+    Card,
+    CardBody,
+    CardHeader,
+    CardTitle,
+    Divider,
+    Flex,
+    Form,
+    FormGroup,
+    Grid,
+    GridItem,
+    Radio,
+    Switch,
+    Title,
+    FormHelperText,
+    HelperText,
+    HelperTextItem,
+} from '@patternfly/react-core';
+import { useFormikContext } from 'formik';
+
+import { ClientPolicy, LifecycleStage } from 'types/policy.proto';
+
+import DownloadCLIDropdown from './DownloadCLIDropdown';
+import {
+    appendEnforcementActionsForAddedLifecycleStage,
+    filterEnforcementActionsForRemovedLifecycleStage,
+    hasEnforcementActionForLifecycleStage,
+} from '../../policies.utils';
+import './PolicyEnforcementForm.css';
+
+function PolicyEnforcementForm() {
+    const { setFieldValue, values } = useFormikContext<ClientPolicy>();
+
+    const hasEnforcementActions =
+        values.enforcementActions?.length > 0 &&
+        !values.enforcementActions?.includes('UNSET_ENFORCEMENT');
+    const [showEnforcement, setShowEnforcement] = useState(hasEnforcementActions);
+
+    function onChangeEnforcementActions(lifecycleStage: LifecycleStage, isChecked: boolean) {
+        const { enforcementActions } = values;
+        setFieldValue(
+            'enforcementActions',
+            isChecked
+                ? appendEnforcementActionsForAddedLifecycleStage(lifecycleStage, enforcementActions)
+                : filterEnforcementActionsForRemovedLifecycleStage(
+                      lifecycleStage,
+                      enforcementActions
+                  ),
+            false // do not validate, because code changes the value
+        );
+    }
+
+    const responseMethodHelperText = showEnforcement
+        ? 'Inform and enforce will execute enforcement behavior at the stages you select.'
+        : 'Inform will always include violations for this policy in the violations list.';
+
+    const hasBuild = values.lifecycleStages.includes('BUILD');
+    const hasDeploy = values.lifecycleStages.includes('DEPLOY');
+    const hasRuntime = values.lifecycleStages.includes('RUNTIME');
+
+    return (
+        <Flex
+            direction={{ default: 'column' }}
+            spaceItems={{ default: 'spaceItemsNone' }}
+            flexWrap={{ default: 'nowrap' }}
+        >
+            <Flex
+                direction={{ default: 'column' }}
+                spaceItems={{ default: 'spaceItemsSm' }}
+                className="pf-v5-u-p-lg"
+            >
+                <Title headingLevel="h2">Enforcement</Title>
+                <div>Select a method to address violations of this policy.</div>
+            </Flex>
+            <Divider component="div" />
```

Comment:
Asking, not necessarily suggesting, whether we dare to consider encapsulating the layout? (In follow up.)
* Although steps 1 to 6 seem the same, **Review** step has side panel.
* Small improvement, especially if we do: replace `div` for subtitle with PatternFly `Text` element, which renders `p` by default.
* Also if we do, move `className="pf-v5-u-p-lg"` prop out of form below

Reply:
With how many steps there are, I could definitely see the value in encapsulating the layout. Seems like an easy way to keep things somewhat consistent across 7 different steps. I've already noticed a few other inconsistencies, but I was trying to find the balance between changing whats necessary to satisfy the requirements while also making improvements along the way 😅